db/model/subject.js

```diff
@@ -101,6 +101,13 @@ module.exports = function subject(seq, dataTypes) {
       type: dataTypes.STRING(constants.fieldlen.longish),
       allowNull: true,
     },
+    sortBy: {
+      type: dataTypes.STRING(constants.fieldlen.longish),  
```

Comment:
On second thought, 4096 is too long for this use case... please add a constant to constants.fieldlen called "sortField" with length 254.